libretroshare/src/jsonapi/jsonapi.h

```diff
@@ -120,40 +120,48 @@ struct JsonApiServer : RsSingleJobThread, p3Config
 	 * @jsonapi{development}
 	 * @return the set of authorized encoded tokens
 	 */
-	std::set<std::string> getAuthorizedTokens();
+	std::map<std::string,std::string> getAuthorizedTokens();
 
 	/**
 	 * @brief Revoke given auth token
 	 * @jsonapi{development}
-	 * @param[in] token decoded
+	 * @param[in] user par of the decoded token
 	 * @return true if the token has been revoked, false otherwise
 	 */
-	bool revokeAuthToken(const std::string& token);
+	bool revokeAuthToken(const std::string& user);
```

Comment:
did you changed the implementation too to revoke user? in that case the method name should be changed too to revokeAuthUser